modules/core/src/main/java/org/apache/ignite/IgniteCache.java

```diff
@@ -1118,6 +1118,23 @@
      */
     public IgniteFuture<V> getAndReplaceAsync(K key, V val);
 
+    /**
+     * Returns the remaining time to live of the specified key.
+     *
+     * @param key Key
+     * @return remaining TTL in milliseconds, null if there is no expiry policy or EternalExpiryPolicy.
```

Comment:
Please specify the behavior of this method in the following cases:
 - the specified `key` as `null` 
 - cache does not contain the specified `key`

perhaps, something like as follows:
```
   * @param key the key whose associated time-to-live value is to be returned
   * 
   * @return the remaining time-to-live, or {@code null}, if it does not exist or there is no expiry policy associated with the key or {@link EternalExpiryPolicy}.
   * @throws IllegalStateException if the cache is {@link #isClosed()}
   * @throws NullPointerException  if the key is null
```